spec/setup_acceptance_node.pp

```diff
@@ -23,6 +23,12 @@
   default: {}
 }
 
+if $facts['os']['name'] == 'CentOS' and $major == '8' {
+  package { 'centos-release-ansible-29':
+    ensure => present,
+  }
+}
```

Comment:
Looking at the `case $major` block, it appears we already assume it's CentOS for version 7 anyway. Perhaps the `package` definition should move to the block on line 19? Then the conditional part can be removed.